docs_src/design/ucr/0002-Device-Parent-Child-Relationships.md

```diff
@@ -0,0 +1,109 @@
+## Device Parent-Child Relationships
+This UCR describes Use Cases for new Device metadata for Parent to Child Relationships for a given Device.
+
+### Submitters
+- Tom Brennan (Eaton)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/800) (2022-07-18)
+
+
+### Market Segments
+Any that deploy EdgeX systems to manage multiple devices.
+In particular, Industrial Gateway systems that connect to multiple south-bound devices
+and provide their data to north-bound services.
+
+### Motivation
+It is frequently important to north-bound services to establish the parent-child relationships
+of the devices found in an EdgeX system. 
+This information is generally used for either protocol data constructs or for display purposes.
+If not know or provided by the south-bound Device Service, this information might be added 
+to the Device metadata by the north-bound or analytic services, or by the user.
+It is desirable that the means of conveying this information become standardized for those systems
+which provide and use it, hence proposing here that there be a common definition and use of this metadata.
+
+### Target Users
+- Product Developers
+- Device Owner
+- Device User
+- Device Maintainer
+- Cloud User
+- Service Provider
+- Software Integrator
+
+### Description
+Some north-bound protocols and some UI designs present the system devices in a hierarchial manner, 
+where it is necessary to know which devices are the parent(s) and which are their children.
+
+These considerations are most important for gateways that are implemented with the EdgeX framework,
+since there are potentially very many south-bound devices connected to a system.
+
+Examples are
```

Comment:
These requirements describe more of a device manager/device vs parent/child need.  EdgeX started down this path before with the concept of a device manager that represented a group of nodes or the "main device" that was responsible for the collection of devices underneath.  Do we need a "grouping" mechanism or do we need a parent/child relationship?  Where the parent/child relationship ever go more than a single layer deep (i.e. would there be grandparent devices)?  Would a child ever have more than one parent?

Reply:
Seems like the owning service is a pretty good grouping mechanism for some devices, and the core-metadata APIs already handle this. Though this doesn't help too much when there are a bunch of services and each has only one device.

Yes to grandparent devices, eg, if the gateway device itself is the top device, and it ingests a device like an energy meter (child) with branch circuit monitoring points (grandchildren).

No to multi-parentage; I can't see that happening.